.enb/lib/page.js

```diff
@@ -0,0 +1,149 @@
+var techs = require('./techs');
+
+module.exports = function(node, options) {
+    var levels = options.levels || [],
+        browsers = options.browser || [],
+        langs = options.langs || [],
+        templateEngine = options.templateEngine || 'BH';
+
+    // essential
+    node.addTechs([
+        [techs.bem.levels, { levels : levels }],
+        [techs.bem.bemjsonToBemdecl, {
+            target : '.tmp.bemdecl.js'
+        }],
+        [techs.bem.deps, {
+            bemdeclFile : '.tmp.bemdecl.js',
+            target : '.tmp.deps.js'
+        }],
+        [techs.bem.files, {
+            depsFile : '.tmp.deps.js'
+        }]
+    ]);
+
+    // css
+    node.addTech([techs.css, {
+        target : '.tmp.css',
+        browsers : browsers
+    }]);
+    node.mode('development', function() {
```

Comment:
I'd rather use `borschik` in both dev and prod modes but in dev without `minify` option.
and I think we can switch `freeze` on by default because without `.borschik` config there won't be any changes and as soon as it appear freezing should be applied.